lib/faexport/scraper.rb

```diff
@@ -614,10 +610,15 @@ def post(path, params)
   def build_submission(elem)
     if elem
       id = elem['id']
-      title_elem = elem.at_css('figcaption') ? elem.at_css('figcaption').at_css('p').at_css('a') : nil
+      title = ""
```

Comment:
Ruby lets you assign a variable from the result of an if statement.  If you add in an `else` block with `""` you can collapse this into a single assignment.  See https://stackoverflow.com/a/13863289

Reply:
That's very neat! Will get that sorted asap